README.md

```diff
@@ -45,33 +45,31 @@ Rector **instantly upgrades and instantly refactors the PHP code of your applica
 
 <br>
 
-## What can Rector do for You?
+## What Can Rector Do for You?
 
 - Rename classes, methods, properties, namespaces or constants
 - Complete [parameter, var or return type declarations](https://www.tomasvotruba.cz/blog/2019/01/03/how-to-complete-type-declarations-without-docblocks-with-rector/) based on static analysis of your code
 - Upgrade your code from PHP 5.3 to PHP 7.4
 - [Migrate your project from Nette to Symfony](https://www.tomasvotruba.cz/blog/2019/02/21/how-we-migrated-from-nette-to-symfony-in-3-weeks-part-1/)
 - [Complete PHP 7.4 property type declarations](https://www.tomasvotruba.cz/blog/2018/11/15/how-to-get-php-74-typed-properties-to-your-code-in-few-seconds/)
-- [Turn Laravel static to Dependency Injection](https://www.tomasvotruba.cz/blog/2019/03/04/how-to-turn-laravel-from-static-to-dependency-injection-in-one-day/)
+- [Refactor Laravel facades to dependency injection](https://www.tomasvotruba.cz/blog/2019/03/04/how-to-turn-laravel-from-static-to-dependency-injection-in-one-day/)
 - And much more...
 
 ...**look at the overview of [all available Rectors](/docs/AllRectorsOverview.md)** with before/after diffs and configuration examples. You can use them to build your own sets.
 
 ## How to Apply Coding Standards?
 
-The AST libraries that Rector uses, don't work well with coding standards, so it's better to let coding standard tools do that.
+The AST libraries that Rector uses aren't well-suited for coding standards, so it's better to let coding standard tools do that.
 
-Your project doesn't have one? Consider adding [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard), [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) or [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer).
+Don't have a coding standard tool for your project? Consider adding [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard), [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) or [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer).
 
 ## Install
 
 ```bash
 composer require rector/rector --dev
 ```
 
-**Do you have conflicts on `composer require` or on run?**
-
-- Use [Docker image](#run-rector-in-docker)!
+**Did you have conflicts during `composer require` or on run? Use the [Docker image](#run-rector-in-docker).**
```

Comment:
Better keep this separated on each line. 

- **bold** = question 
- under it = answer

Reply:
What do you think about quote-formatting this section? The single bullet point looked a little out of place to me.
e.g.
> **Did you have conflicts during `composer require` or on run?**  
> Use the [Docker image](#run-rector-in-docker).

Reply:
Personally I don't read quotes in README. I'd expect them in posts